src/Inventory/Asset/NetworkPort.php

```diff
@@ -443,6 +444,26 @@ private function handleVlans(\stdClass $port, int $netports_id)
         }
     }
 
+    private function handleMetrics(\stdClass $port, int $netports_id)
+    {
+        $input = (array)$port;
+        //only update networkport metric if needed
+        if (
+            isset($input['ifinbytes'])
+            && isset($input['ifoutbytes'])
+            && isset($input['ifinerrors'])
+            && isset($input['ifouterrors'])
+        ) {
```

Comment:
You can use a shorter notation here.
```suggestion
        if (isset($input['ifinbytes'], $input['ifoutbytes'], $input['ifinerrors'], $input['ifouterrors'])) {
```